perun-core/src/main/java/cz/metacentrum/perun/core/impl/UsersManagerImpl.java

```diff
@@ -1406,4 +1456,37 @@ public void deleteSponsorLinks(PerunSession sess, User sponsor) {
 		}
 	}
 
+	private String getSQLSelectForUsersPage(UsersPageQuery query) {
+		String select =
+			"SELECT " + userMappingSelectQuery +
+				" ,count(*) OVER() AS total_count" +
+				" FROM users";
+
+		String selectWithMembers =
+			"SELECT " + userMappingSelectQuery +
+				" ,count(*) OVER() AS total_count" +
+				" FROM users LEFT JOIN members on members.user_id = users.id";
+
+
+		return !query.isWithoutVo() && Objects.equals(query.getSearchString(), "") ? select : selectWithMembers;
```

Comment:
We can use `isEmpty(query.getSearchString()))` here as well.

Reply:
Fixed.